### ISSUE TITLE
Please support nextcloud 30

### ISSUE THREAD

**Issue by svedbox** [matteo-convertino/otpmanager-nextcloud#59]
Please, we very wait it.

**Comment by Toutatux** [comment]
Hey ! Yeah, please !!!

**Comment by pogmommy** [comment]
#53 mentioned that otpmanager works with Nextcloud 30. After testing myself, I can confirm

**Comment by sabaka1** [comment]
yeah kinda. BUT you cant share 2fa with other users on NC30

**Comment by BerziOnline** [comment]
> yeah kinda. BUT you cant share 2fa with other users on NC30

Exactly, I have the same issue here - cannot share with other users. The OTP is appearing, but if they are trying to unlock by the initial sharing password is is loading for ever. Tried that with different users and password for 100% correct. 
Sharing doesn't work.

**Comment by gh-repo-status-check** [comment]
Yeah I think @matteo-convertino will no longer  maintain this project :(  - there is no commit since 6 months

**Comment by JoshuaPettus** [comment]
Anyone tried it with NC31 release yet to see if it still works?  
[EDIT]
Just snapshot-ed my setup and gave it a try.  Core functionality works, as with the phone app.  Can't speak to the OTP sharing issue mentioned as I'm a single user :(

**Comment by matteo-convertino** [comment]
I am closing this issue since the [new release](https://github.com/matteo-convertino/otpmanager-nextcloud/releases/tag/v1.0.1) supports Nextcloud 32 🚀
